lms/static/js/spec/edxnotes/views/tabs/search_results_spec.js

```diff
@@ -195,20 +188,140 @@ define([
+            Helpers.verifyPageData(view, this.tabsCollection, tabInfo, searchResultsTabId, notes);
+
+            view.$('input#page-number-input').val('2');
+            view.$('input#page-number-input').trigger('change');
+            Helpers.verifyRequestParams(
+                requests[requests.length - 1].url,
+                {page: '2', page_size: '10'}
+            );
+
+            notes = Helpers.createNotesData(
+                {
+                    numNotesToCreate: 2,
+                    count: 12,
+                    num_pages: 2,
+                    current_page: 2,
+                    start: 10
+                }
+            );
+            Helpers.respondToRequest(requests, notes, true);
+            Helpers.verifyPaginationInfo(view, "Showing 11-12 out of 12 total", 2, 2);
+            Helpers.verifyPageData(view, this.tabsCollection, tabInfo, searchResultsTabId, notes);
+        });
+
+        it("can navigate forward and backward", function () {
+            var requests = AjaxHelpers.requests(this),
+                page1Notes = Helpers.createNotesData(
+                    {
+                        numNotesToCreate: 10,
+                        count: 15,
+                        num_pages: 2,
+                        current_page: 1,
+                        start: 0
+                    }
+                ),
+            view = getView(this.tabsCollection);
+
+            submitForm(view.searchBox, 'awesome');
+            Helpers.respondToRequest(requests, page1Notes, true);
+            Helpers.verifyPaginationInfo(view, "Showing 1-10 out of 15 total", 1, 2);
+            Helpers.verifyPageData(view, this.tabsCollection, tabInfo, searchResultsTabId, page1Notes);
+
+            view.$('.pagination .next-page-link').click();
+            Helpers.verifyRequestParams(
+                requests[requests.length - 1].url,
+                {page: '2', page_size: '10'}
+            );
+            var page2Notes = Helpers.createNotesData(
+                {
+                    numNotesToCreate: 5,
+                    count: 15,
+                    num_pages: 2,
+                    current_page: 2,
+                    start: 10
+                }
+            );
+            Helpers.respondToRequest(requests, page2Notes, true);
+            Helpers.verifyPaginationInfo(view, "Showing 11-15 out of 15 total", 2, 2);
+            Helpers.verifyPageData(view, this.tabsCollection, tabInfo, searchResultsTabId, page2Notes);
+
+            view.$('.pagination .previous-page-link').click();
+            Helpers.verifyRequestParams(
+                requests[requests.length - 1].url,
+                {page: '1', page_size: '10'}
+            );
+            Helpers.respondToRequest(requests, page1Notes);
+
+            Helpers.verifyPaginationInfo(view, "Showing 1-10 out of 15 total", 1, 2);
+            Helpers.verifyPageData(view, this.tabsCollection, tabInfo, searchResultsTabId, page1Notes);
```

Comment:
A lot of the logic in these tests is the same as the logic in the new tests in `recent_activity_spec.js` Can we not make helper functions and just pass in the different parameters for the two views?